### ISSUE TITLE
Evaluate logging frameworks for better log formatting for local and cluster environments

### ISSUE THREAD

**Issue by decimalator** [osulp/Scholars-Archive#2337]
### Descriptive summary

Our logging configuration is all over the place at the moment. Production doesn't adjust logging very much, but Staging and Development use `lograge`.

We need more structured logging to make log processing easier. We should either:

- configure production to use `lograge` to structure our logs for easier ingest
- remove `lograge` and configure the standard Rails logger to structure logs for easier ingest
- find another logging library to structure logs for easier ingest

Formatting our logs to be more structured will allow us to more easily extract relevant information from the logs. This will allow us to label certain log data to make queries for that data faster. The format that we use doesn't matter as long as we can easily extract certain info from them:

- work id
- user id
- collection id
- fileset id
etc.

**Comment by decimalator** [comment]
examples:

Production now:

```
D, [2022-02-22T10:53:44.039525 #259] DEBUG -- : [fef86dc6e968ea8dd526d3a6c5cda873]   Load LDP (13.6ms) http://fcrepo.sa-prod.svc.cluster.local:8080/fcrepo/rest/prod/ae/27/66/7e/ae27667e-76de-48f8-8bca-a72d98117dab/1a/62/f0/42/1a62f042-4751-4d55-9018-4c9cda30c403 Service: 47150069646780
```

JSON:

```
{
  "date": "2022-02-22T10:53:44.039525",
  "log_level": "DEBUG",
  "req_id": "fef86dc6e968ea8dd526d3a6c5cda873",
  "uri": "http://fcrepo.sa-prod.svc.cluster.local:8080/fcrepo/rest/prod/ae/27/66/7e/ae27667e-76de-48f8-8bca-a72d98117dab/1a/62/f0/42/1a62f042-4751-4d55-9018-4c9cda30c403",
  "message": "Load LDP (13.6ms) ",
  "service_id": "47150069646780"
}
```

Key=Value:
```
[2022-02-22T10:53:44.039525] level=DEBUG req_id=fef86dc6e968ea8dd526d3a6c5cda873 uri="http://fcrepo.sa-prod.svc.cluster.local:8080/fcrepo/rest/prod/ae/27/66/7e/ae27667e-76de-48f8-8bca-a72d98117dab/1a/62/f0/42/1a62f042-4751-4d55-9018-4c9cda30c403" service_id=47150069646780 message="Load LDP (13.6ms)"
```

**Comment by lamtu1** [comment]
Fixed by PR #2423
